extensionsGallery-insider.json

```diff
@@ -3603,8 +3603,8 @@
 					},
 					"versions": [
 						{
-							"version": "0.1.0",
-							"lastUpdated": "9/1/2020",
+							"version": "0.1.1",
```

Comment:
You need to change the links below in these files too

Reply:
Made the url change as well.